### ISSUE TITLE
poor accuracy on svt1 images

### ISSUE THREAD

**Issue by abuvaneswari** [TJCVRS/CRNN_Tensorflow#22]
Dear author,

My test data set consists of text-localized images from the svt1 dataset (http://tc11.cvc.uab.es/datasets/SVT_1). The localization was done using Seglink (https://github.com/bgshih/seglink). 

Here is the flow:
svt1 images -----> seglink ----> CRNN_Tensorflow

I utilized the checkpoint at step 19999  that you have provided under models directory for prediction. The prediction error turns out to be very bad.

Can you please share the log / plot of your loss and validation dataset accuracy (/distance) at each epoch during your training? I would like to see if there was overfitting. Also, can you please make your checkpoint 9999 and 29999 available?

thanks a lot!
Best,
Buvana

**Comment by pribadihcr** [comment]
@abuvaneswari,

I am interesting with seglink. Can you share the pretrained model please?. Thanks

**Comment by abuvaneswari** [comment]
@pribadihcr , actually, I used the following repo for seglink:
https://github.com/dengdan/seglink

Excellent results and the repo comes with checkpoints.

**Comment by pribadihcr** [comment]
@abuvaneswari, Oh great. thanks

**Comment by TJCVRS** [comment]
@abuvaneswari I have met the same overfit problem and I have not found a suitable solution for the problem you are welcomed to share any idea that may overcome the problem
